src/menu_new_game/A5CC50.c

```diff
@@ -8,78 +8,358 @@ extern Sprite D_80117F98_A93348; // card
 extern Sprite D_80109880_A84C30; // background
 
 char* D_800E2FF0[] = {
-    "あ", "い", "う", "え", "お",
-    "か", "き", "く", "け", "こ",
-    "さ", "し", "す", "せ", "そ",
-    "た", "ち", "つ", "て", "と",
-    "な", "に", "ぬ", "ね", "の",
-    "は", "ひ", "ふ", "へ", "ほ",
-    "ま", "み", "む", "め", "も",
-    "や", "ゆ", "よ", "　", "　",
-    "ら", "り", "る", "れ", "ろ",
-    "わ", "を", "ん", "　", "　",
-    "が", "ぎ", "ぐ", "げ", "ご",
-    "ざ", "じ", "ず", "ぜ", "ぞ",
-    "だ", "ぢ", "づ", "で", "ど",
-    "ば", "び", "ぶ", "べ", "ぼ",
-    "ぱ", "ぴ", "ぷ", "ぺ", "ぽ",
-    "ぁ", "ぃ", "ぅ", "ぇ", "ぉ",
-    "っ", "ゃ", "ゅ", "ょ", "\xA1\xDD",
-    "ア", "イ", "ウ", "エ", "オ",
-    "カ", "キ", "ク", "ケ", "コ",
-    "サ", "シ", "ス", "セ", "ソ",
-    "タ", "チ", "ツ", "テ", "ト",
-    "ナ", "ニ", "ヌ", "ネ", "ノ",
-    "ハ", "ヒ", "フ", "ヘ", "ホ",
-    "マ", "ミ", "ム", "メ", "モ",
-    "ヤ", "ユ", "ヨ", "　", "　",
-    "ラ", "リ", "ル", "レ", "ロ",
-    "ワ", "ヲ", "ン", "　", "　",
-    "ガ", "ギ", "グ", "ゲ", "ゴ",
-    "ザ", "ジ", "ズ", "ゼ", "ゾ",
-    "ダ", "ヂ", "ヅ", "デ", "ド",
-    "バ", "ビ", "ブ", "ベ", "ボ",
-    "パ", "ピ", "プ", "ペ", "ポ",
-    "ァ", "ィ", "ゥ", "ェ", "ォ",
-    "ッ", "ャ", "ュ", "ョ", "\xA1\xDD",
-
-    "Ａ", "Ｂ", "Ｃ", "Ｄ", "Ｅ",
-    "Ｆ", "Ｇ", "Ｈ", "Ｉ", "Ｊ",
-    "Ｋ", "Ｌ", "Ｍ", "Ｎ", "Ｏ",
-    "Ｐ", "Ｑ", "Ｒ", "Ｓ", "Ｔ",
-    "Ｕ", "Ｖ", "Ｗ", "Ｘ", "Ｙ",
-    "Ｚ", "　", "　", "　", "　",
-    "ａ", "ｂ", "ｃ", "ｄ", "ｅ",
-    "ｆ", "ｇ", "ｈ", "ｉ", "ｊ",
-    "ｋ", "ｌ", "ｍ", "ｎ", "ｏ",
-    "ｐ", "ｑ", "ｒ", "ｓ", "ｔ",
-    "ｕ", "ｖ", "ｗ", "ｘ", "ｙ",
-    "ｚ", "　", "　", "　", "　",
-    "０", "１", "２", "３", "４",
-    "５", "６", "７", "８", "９",
-    "！", "？", "、", "。", "・",
-    "\xA1\xDD", "\xA1\xC1", "　", "　", "　",
-    "　", "　", "　", "　", "　",
-
-    "Ａ", "Ｂ", "Ｃ", "Ｄ", "Ｅ",
-    "Ｆ", "Ｇ", "Ｈ", "Ｉ", "Ｊ",
-    "Ｋ", "Ｌ", "Ｍ", "Ｎ", "Ｏ",
-    "Ｐ", "Ｑ", "Ｒ", "Ｓ", "Ｔ",
-    "Ｕ", "Ｖ", "Ｗ", "Ｘ", "Ｙ",
-    "Ｚ", "ａ", "ｂ", "ｃ", "ｄ",
-    "ｅ", "ｆ", "ｇ", "ｈ", "ｉ",
-    "ｊ", "ｋ", "ｌ", "ｍ", "ｎ",
-    "ｏ", "ｐ", "ｑ", "ｒ", "ｓ",
-    "ｔ", "ｕ", "ｖ", "ｗ", "ｘ",
-    "ｙ", "ｚ", "０", "１", "２",
-    "３", "４", "５", "６", "７",
-    "８", "９", "†", "‡", "¶",
-    "\xA1\xC1", "！", "？", "＠", "＃",
-    "％", "＆", "＊", "（", "）",
-    "＿", "＋", "＝", "ー", "’",
-    "”", "；", "：", "／", "＄",
-    "．", "，", "＜", "＞", "♂",
-    "♀", "［", "］", "ε", "　",
+    "あ",
```

Comment:
this isn't ideal

Reply:
This is gonna need `clang-format off` fences, due to the manual formatting.